src/extra.ml

```diff
@@ -136,6 +157,37 @@ module IntMap = Map.Make(Int)
 (* Functional maps with [string] keys. *)
 module StrMap = Map.Make(String)
 
+(** Position of a subterm of a term. *)
+module Position =
```

Comment:
Why not calling it Subterm to avoid confusion with Pos?

Reply:
done in cf741e5